PWGGA/GammaConv/AliAnalysisTaskGammaCaloMergedML.cxx

```diff
@@ -2793,12 +2817,10 @@ void AliAnalysisTaskGammaCaloMergedML::ProcessTrueClusterCandidatesAOD(AliAODCon
 	  }
 	  tTrueMergedCaloClusterPi0[fiCut]->Fill();
 	  ResetBuffer();
-        }if (motherPDG == 221){//Eta
+        } if (motherPDG == 221){//Eta
```

Comment:
here is a good example where instead of checking all these different cases one could just have one tree and save motherPDG